pkgs/by-name/ku/kubectl-kcl/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "kubectl-kcl";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "kcl-lang";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-yuNQSO1xQCb5H55mOUTVrojeWWkDOmAGJIzUs6qCWO4=";
+  };
+
+  vendorHash = "sha256-GD4C4jlxVMpJ/bhpQ3VDkBMBBQkXyhMMga+WhVdvI/I=";
+
+  ldflags = [
+    "-X kcl-lang.io/kubectl-kcl/cmd.Version=${version}"
+  ];
+
+  meta = with lib; {
```

We prefer not to do this, but its a bit of a nit
```suggestion
  meta = {
```

---

amended